tdrs-frontend/src/actions/requestAccess.js

```diff
@@ -0,0 +1,33 @@
+// import axios from 'axios'
+import { SET_AUTH } from './auth'
+import utils from '../utils'
+
+export const PATCH_REQUEST_ACCESS = 'PATCH_REQUEST_ACCESS'
+export const SET_REQUEST_ACCESS = 'SET_REQUEST_ACCESS'
+export const SET_REQUEST_ACCESS_ERROR = 'SET_REQUEST_ACCESS_ERROR'
+export const CLEAR_REQUEST_ACCESS = 'CLEAR_REQUEST_ACCESS'
+
+export const requestAccess = ({ firstName, lastName, stt: { id } }) => async (
+  dispatch
+) => {
+  dispatch({ type: PATCH_REQUEST_ACCESS })
+  try {
+    const URL = `${process.env.REACT_APP_BACKEND_URL}/users/set_profile/`
+    const user = { first_name: firstName, last_name: lastName, stt: { id } }
+    const { data } = await (await utils.axiosInstance).patch(URL, user, {
```

Comment:
We are using a custom axios client for these authenticated patch requests, we set the csrf token as a default value here, but not for the global axios client.